streaming-ai/src/main/java/com/datastax/oss/streaming/ai/completions/OpenAICompletionService.java

```diff
@@ -66,8 +58,25 @@ public ChatCompletions getChatCompletions(
     return result;
   }
 
+  private ChatRequestMessage getChatRequestMessage(ChatMessage message) {
+    switch (message.getRole()) {
+      case "system":
+        return new ChatRequestSystemMessage(message.getContent());
+      case "assistant":
+        return new ChatRequestAssistantMessage(message.getContent());
+      case "user":
+        return new ChatRequestUserMessage(message.getContent());
+      case "function":
+        return new ChatRequestFunctionMessage("", message.getContent());
+      case "tool":
+        return new ChatRequestToolMessage(message.getContent(), "");
+      default:
+        return new ChatRequestMessage();
```

Comment:
should we throw error here ? it seems we're losing the content

Reply:
Yes, an error would be better, a message without any role or invalid role should throw an error, as there is no class which could accept the such a message.